cmd/shell.go

```diff
@@ -98,13 +98,10 @@ func (log grpcLogger) Fatalln(args ...interface{}) {
 	os.Exit(1)
 }
 func (log grpcLogger) Print(args ...interface{}) {
-	log.AuditErr(fmt.Sprint(args...))
```

Comment:
Removing these breaks all of the Fatal methods above as they call them in order to log messages. Also are you sure grpc doesn't use these to print non-fatal error messages? Seems like we could loose some useful debugging information by killing these.